### PR TITLE
Feature/add new booking

### PR DESCRIPTION
**Pull request by LKessell**
## Reason for Pull Request:
Add a selected room as a booking and POST to the api

## Testing Method(s)
[ ] Terminal/Mocha
[x] localhost
[ ] Lighthouse

## Changes Implemented:
- Add a modal to confirm a room booking
- Ability to select a room and POST booking data to api
- Updated data is shown on the dashboard
- Success message shown on dashboard after posting

### CODE REVIEW

**Comment by LKessell** [comment]
@niksseif I asked for Kat's review on this PR

File: src/domUpdates.js [hunk 1]
```diff
@@ -44,24 +44,26 @@ const domUpdates = {
...
     element.innerText = text;
   },
 
   renderRoomCards(container, data) {
     container.innerHTML = '';
+
     if (data.length) {
       data.forEach(element => {
-        let image = element.roomType.split(' ')[0];
+        const image = element.roomType.split(' ')[0];
```
Comment (k-atwhite) [inline src/domUpdates.js:56]: This is such a clever little bit of code to generate your images!

File: src/scripts.js [hunk 2]
```diff
@@ -56,26 +60,54 @@ dropdownButton.addEventListener('click', () => {
...
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify(bookingInfo)
+  })
+    .then(checkForPostError)
+    .then(setUpRooms)
+    .then(resetToDashboard)
+    .catch(err => displayErrorMesssage(err));
+}
+
 const checkForGetError = (response) => {
```
Comment (k-atwhite) [inline src/scripts.js:111]: eff i forgot I need to add one of these

File: src/css/base.scss [hunk 3]
```diff
@@ -239,6 +239,45 @@ h3 {
   margin-bottom: 2vw;
 }
 
+.modal-container {
+  display: flex;
```
Comment (k-atwhite) [inline src/css/base.scss:243]: i bet this flex - centering could be turned into a sass variable

**Review by k-atwhite — APPROVED** [review]
As usually, i have very little to review. I think you have quite a few clever little sections. 
Good job working from mobile, outward!

**Pull request merged**